### PR TITLE
Replaced Chef on Ansible cartridge, Docker and AMIs update

### PR DESCRIPTION
**Pull request by anton-kasperovich**
What has been done as part of this PR:

* Replaced Chef on Ansible cartridge - https://github.com/Accenture/adop-cartridge-ansible
* Updated AMIs, latest stable CentOS 7 (2018-Jan-14) release
* Added new AMIs to support following regions: eu-west-2, eu-west-3, ca-central-1, ap-northeast-1
* Updated Docker version from 1.10.3 to 17.05.0
* Updated Docker Compose version from 1.7.1 to 1.16.1
* Removed Chef Platform extension from extension collection JSON

### CODE REVIEW

**Comment by RobertNorthard** [comment]
Test cases performed:

**Scenario 1**: Spin up DOA
- [/] No errors in cloud init
- [/]

File: Standing_Up_Tools/doa_stack.json [hunk 1]
```diff
@@ -210,7 +223,7 @@
                                 "cd /data/adop-docker-compose\n",
                                 "./adop workspace -w DOA create; sleep 10\n",
                                 "./adop project -p Labs -w DOA create; sleep 10\n",
-                                "./adop project -p Labs -w DOA load_collection -c https://raw.githubusercontent.com/Accenture/adop-doa-materials/master/Cartridge_Collections/doa_cartridge_collection.json\n",
+                                "./adop project -p Labs -w DOA load_collection -c https://raw.githubusercontent.com/anton-kasperovich/adop-doa-materials/chef_to_ansible_replacement/Cartridge_Collections/doa_cartridge_collection.json\n",
```
Comment (RobertNorthard) [inline Standing_Up_Tools/doa_stack.json]: Shouldn't this be under the Accenture namespace?

**Comment by RobertNorthard** [comment]
Looks great to me thanks @anton-kasperovich.

**Comment by anton-kasperovich** [comment]
@RobertNorthard decision was made to drop default support of Chef, although anyone can just load Chef cartridge or/and Platform Extension separately via Load_Cartridge job for instance.

**Comment by RobertNorthard** [comment]
Okay, thanks. Merging. Chef can always be added back to cartridge collection if required to make it optional.

**Pull request merged**